theano/compile/profiling.py

```diff
@@ -1245,16 +1243,6 @@ def print_stats(stats1, stats2):
 
             print("---", file=file)
 
-        if (hasattr(theano, 'sandbox') and
```

Comment:
There is no way to do it currently, but it could be added.  See https://github.com/Theano/libgpuarray/issues/400